scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -3327,6 +3370,7 @@ bool run_timer_cb(int key)
 struct layered_timer layered_timers[MAX_TIMERS] = {
 	{15LLU * NSEC_PER_SEC, CLOCK_BOOTTIME, 0},
 	{1LLU * NSEC_PER_SEC, CLOCK_BOOTTIME, 0},
+	{1LLU * NSEC_PER_MSEC, CLOCK_BOOTTIME, 0},
```

Comment:
this was chosen arbitrarily, should probably be more configurable but makes working with timers a little annoying